nodejs/week2/app.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const app = express();
+const port = process.env.PORT || 3000;
+const doc = require('./documents.json');
+// const knex = require("./database")
+// Support parsing JSON requests
+app.use(express.json()); //to accept json request
+
+app.get('/', (req, res) => {
+  res.send('This is a search engine');
+});
+
+// It does not work
+app.get('/search', (req, res) => {
```

Comment:
It doesn't work for me too, I don't understand why

Reply:
Do you mean Vie's code does not work? What error(s) are you seeing?

Reply:
It's magic, today it works)))). Yesterday I received HTML in my body, but it was my mistake.
![image](https://user-images.githubusercontent.com/73549460/193309854-21c7fbb4-bd07-4528-b2a2-54f9c48da4fc.png)